### PR TITLE
[TRAFODION-3094] Add Syntax and Examples for *DELETE Statement* in *Trafodion Reference Manual*

### PR DESCRIPTION
**Pull request by liuyu000**

### CODE REVIEW

File: docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc [hunk 1]
```diff
@@ -4239,23 +4239,44 @@ For the searched DELETE form, if no WHERE clause exists, all rows are deleted fr
...
 [[delete_syntax]]
 === Syntax Description of DELETE
 
-* `_table_`
+* `_table-name_`
 +
 names the user table or view from which to delete rows. _table_ must be a base table or an updatable view. To refer to a
 table or view, use the ANSI logical name. See <<database_object_names,Database Object Names>>.
 
+* `WITHOUT NO ROLLBACK`
```
Comment (DaveBirdsall) [inline docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc]: This should be "WITH NO ROLLBACK"

File: docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc [hunk 2]
```diff
@@ -4239,23 +4239,44 @@ For the searched DELETE form, if no WHERE clause exists, all rows are deleted fr
...
++
+performs non-transactional delete.
```
Comment (DaveBirdsall) [inline docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc]: You may want to explain what a non-transactional delete is (unless it's explained somewhere else and I missed it). A non-transactional delete is one that happens outside a transaction. That means that if the statement is interrupted, any deletes that have been done are still permanent.

File: docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc [hunk 3]
```diff
@@ -4342,6 +4363,45 @@ DELETE FROM invent.partsupp WHERE suppnum NOT IN
...
+WHERE id<30;
+```
+
+* This example shows that the option `WITH NO ROLLBACK` is only supported if there is no
+index on the table.
++
+```
+SQL>DELETE WITH NO ROLLBACK FROM test;
+
+*** ERROR[3232] INSERT, UPDATE, or DELETE of table TRAFODION.SEABASE.TEST with dependent Index : TRAFODION.SEABASE.INDEX_TEST is not allowed when NO ROLLBACK transaction setting is in effect. [2018-05-30 05:42:47]
```
Comment (DaveBirdsall) [inline docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc]: The error message looks a bit rough but that's an issue for the code, not the manual. We can fix that later.

File: docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc (hunk 1 above)
Comment (liuyu000) [reply docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc]: Oops  :stuck_out_tongue_closed_eyes: thanks Dave, I've fixes that :blush:

File: docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc (hunk 2 above)
Comment (liuyu000) [reply docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc]: Thanks Dave, I've added this explanation :raising_hand:

File: docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc (hunk 3 above)
Comment (liuyu000) [reply docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc]: Should I delete or keep the example?

**Comment by sandhyasun** [comment]
+1

**Comment by liuyu000** [comment]
@sandhyasun Sorry for the late reply, I just saw your comment, thanks 😁

File: docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc (hunk 3 above)
Comment (DaveBirdsall) [reply docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc]: Keeping it is fine.

**Pull request merged**